xml/System.Reflection/StrongNameKeyPair.xml

```diff
@@ -130,6 +130,7 @@
         <exception cref="T:System.ArgumentNullException">
```

Comment:
This type is Obsoleted, not sure if we need to add anything about that

Reply:
Once we update the APIs for preview 4, that info should get imported (it's not something we add manually).